lib/jenkins_pipeline_builder/cli/helper.rb

```diff
@@ -71,23 +76,32 @@ def self.valid_cli_creds?(options)
       end
 
       def self.process_creds_file(file)
-        return JSON.parse(IO.read(File.expand_path(file))) if file.end_with? 'json'
-        YAML.load_file(File.expand_path(file))
+        return load File.expand_path(file) if file.end_with? 'rb'
+        return self.jenkins_api_creds = JSON.parse(IO.read(File.expand_path(file))) if file.end_with? 'json'
+        self.jenkins_api_creds = YAML.load_file(File.expand_path(file))
       end
 
       def self.process_cli_creds(options)
-        creds = {}.with_indifferent_access.merge options
-        if creds[:server] =~ Resolv::AddressRegex
-          creds[:server_ip] = creds.delete :server
-        elsif creds[:server] =~ URI.regexp
-          creds[:server_url] = creds.delete :server
+        self.jenkins_api_creds = {}.with_indifferent_access.merge options
+        if jenkins_api_creds[:server] =~ Resolv::AddressRegex
+          jenkins_api_creds[:server_ip] = jenkins_api_creds.delete :server
+        elsif jenkins_api_creds[:server] =~ URI.regexp
+          jenkins_api_creds[:server_url] = jenkins_api_creds.delete :server
         else
-          msg = "server given (#{creds[:server]}) is neither a URL nor an IP."
+          msg = "server given (#{jenkins_api_creds[:server]}) is neither a URL nor an IP."
           msg << ' Please pass either a valid IP address or valid URI'
           $stderr.puts msg
           exit 1
         end
-        creds
+      end
+
+      private_class_method
+
+      def self.default_file
+        found_suffix = DEFAULT_FILE_FORMATS.find do |suffix|
```

Comment:
What's the intended behavior when there are login.rb and login.yml files? This will always return in the order the array is defined. Should this look for all of them and raise an error if there is more than one?